README.md

```diff
@@ -192,9 +191,6 @@ This project was kindly sponsored by [nearForm](http://nearform.com).
 This library is heavily inspired by Richard Rodger's
 [patrun](http://npm.im/patrun) and [Seneca](http://npm.im/seneca).
 
-Bloomrun would not be possible without
-[bloomfilter](https://www.npmjs.com/package/bloomfilter).
-
 The bloomrun logo was created, with thanks, by [Dean McDonnell](https:/github.com/mcdonnelldean)
```

Comment:
efforts made to use "BloomRun" - but not here - should this be "The BloomRun logo...."